styles/variables.less

```diff
@@ -4,3 +4,5 @@
 // App
 // ---------------------------------------
 @agate-app-keepout: 18px;
+
+@agate-panels-bg-color: transparent;
```

Comment:
Shouldn't this be in a color file vs the variables file?

Reply:
You're absolutely right. This seems different than when I saw it. Maybe I just didn't notice. This definitely belongs in each of the `colors-*.less` files, for the same reasons explained in https://github.com/enactjs/agate/pull/141#discussion_r327225227.

Reply:
I also didn't notice that I had to define color variables in color-*.less files. I fixed it.